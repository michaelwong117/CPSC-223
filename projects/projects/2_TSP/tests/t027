#!/bin/bash
# Repeated methods

trap "/usr/bin/killall -q -u $USER ./TSP 2>/dev/null" 0 1 2 3 9 15
trap "/bin/rm -f $STDERR" 0 1 2 3 9 15
if [ ! -x ./TSP ]; then
  echo './TSP is missing or not executable'
  echo './TSP is missing or not executable' 1>&2
  exit 1
fi


/c/cs474/bin/run -stderr=/dev/null ./TSP /c/cs223/hw2/Tests/t027.in -given -exchange adjacent -farthest -exchange any -exchange any -exchange adjacent -given < /dev/null
