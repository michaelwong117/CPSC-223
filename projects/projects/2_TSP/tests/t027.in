6
DEN DME LOS MEL DAC BOG
39.861667 -104.673056
55.408611 37.906389
6.577500 3.321111
-37.673333 144.843333
23.843333 90.397778
4.701667 -74.146944
