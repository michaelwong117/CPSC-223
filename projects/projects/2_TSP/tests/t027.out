-given            :     60184.29 DEN DME LOS MEL DAC BOG DEN
-exchange adjacent:     48275.21 DEN MEL DAC DME LOS BOG DEN
-farthest         :     59095.53 DEN LOS BOG DME DAC MEL DEN
-exchange any     :     48275.21 DEN MEL DAC DME LOS BOG DEN
-exchange any     :     48275.21 DEN MEL DAC DME LOS BOG DEN
-exchange adjacent:     48275.21 DEN MEL DAC DME LOS BOG DEN
-given            :     60184.29 DEN DME LOS MEL DAC BOG DEN
